core/templates/dev/head/components/playlist_tile/collection_playlist_tile_directive.html

```diff
@@ -0,0 +1,41 @@
+<md-card class="oppia-activity-summary-tile oppia-activity-playlist-tile">
```

Comment:
I have deleted these file! :dagger: Now I am reusing the existing tile and customizing only slightly.